### ISSUE TITLE
[feat] auto-executing steps

### ISSUE THREAD

**Issue by pkarw** [callstackincubator/cali#27]
Is there a plan for the suggested command line tools to be automatically executed by the tool (after ofc. accepting it by the user?) I mean:
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/a9568521-ac8b-4ab1-a070-b319cca1e34f">

OK, I can run it no problem, but a little bit ux-breaking to switch between the tabs

**Comment by grabbou** [comment]
Yeah, I think we should just add `execSync` tool, although prepend it with a notice that it must be always confirmed and let it accept "confirmedByUser" boolean.
